test/JDBC/expected/securityadmin_role-vu-verify.out

```diff
@@ -0,0 +1,1520 @@
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: User does not have permission to perform this action.)~~
+
+
+Alter server role securityadmin add member no_securityadmin_login1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Current login no_securityadmin_login1 does not have permission to alter server role)~~
+
+
+Alter server role securityadmin drop member no_securityadmin_login1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Current login no_securityadmin_login1 does not have permission to alter server role)~~
+
+
+-- only current login and fixed server roles
+SELECT name, type, type_desc, default_database_name, default_language_name, credential_id, owning_principal_id, is_fixed_role
+FROM sys.server_principals ORDER BY name
+GO
+~~START~~
+varchar#!#char#!#nvarchar#!#varchar#!#varchar#!#int#!#int#!#bit
+jdbc_user#!#S#!#SQL_LOGIN#!#master#!#English#!#-1#!#-1#!#0
+no_securityadmin_login1#!#S#!#SQL_LOGIN#!#master#!#English#!#-1#!#-1#!#0
+public#!#R#!#SERVER_ROLE#!#<NULL>#!#<NULL>#!#<NULL>#!#1#!#0
+securityadmin#!#R#!#SERVER_ROLE#!#<NULL>#!#English#!#<NULL>#!#1#!#1
+sysadmin#!#R#!#SERVER_ROLE#!#<NULL>#!#English#!#<NULL>#!#1#!#1
+~~END~~
+
+
+-- only current login
+select name, type, usage from sys.login_token order by name;
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar
+no_securityadmin_login1#!#SQL LOGIN#!#GRANT OR DENY
+~~END~~
+
+
+-- terminate-tsql-conn user=no_securityadmin_login1 password=123
+
+-- tsql user=securityadmin_login1 password=123
+-- unsupported
+REVOKE SHOWPLAN FROM no_securityadmin_user1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: 'REVOKE Database' is not currently supported in Babelfish)~~
+
+
+REVOKE CREATE VIEW FROM no_securityadmin_user1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: 'REVOKE Database' is not currently supported in Babelfish)~~
+
+
+-- System objects
+-- All rows of server_prinicipals view should be visible to securityadmin login
+-- limited rows should get displayed
+SELECT name, type, type_desc, default_database_name, default_language_name, credential_id, owning_principal_id, is_fixed_role
+FROM sys.server_principals 
+WHERE name =  'jdbc_user';
+GO
+~~START~~
+varchar#!#char#!#nvarchar#!#varchar#!#varchar#!#int#!#int#!#bit
+jdbc_user#!#S#!#SQL_LOGIN#!#master#!#English#!#-1#!#-1#!#0
+~~END~~
+
+
+SELECT name, type, type_desc, default_database_name, default_language_name, credential_id, owning_principal_id, is_fixed_role
+FROM sys.server_principals 
+WHERE name =  'sysadmin';
+GO
+~~START~~
+varchar#!#char#!#nvarchar#!#varchar#!#varchar#!#int#!#int#!#bit
+sysadmin#!#R#!#SERVER_ROLE#!#<NULL>#!#English#!#<NULL>#!#1#!#1
+~~END~~
+
+
+SELECT name, type, type_desc, default_database_name, default_language_name, credential_id, owning_principal_id, is_fixed_role
+FROM sys.server_principals 
+WHERE name =  'public';
+GO
+~~START~~
+varchar#!#char#!#nvarchar#!#varchar#!#varchar#!#int#!#int#!#bit
+public#!#R#!#SERVER_ROLE#!#<NULL>#!#<NULL>#!#<NULL>#!#1#!#0
+~~END~~
+
```

Comment:
nit: There are multiple such queries starting from 'jdbc_user' to '%securityadmin%'. Let's just combine them to save some test execution time.